imbi/endpoints/operations_log.py

```diff
@@ -166,3 +174,32 @@ class RecordRequestHandler(_RequestHandlerMixin,
                ticket_slug = %(ticket_slug)s,
                version = %(version)s
          WHERE id = %(id)s""")
+
+    async def delete(self, *args, **kwargs):
+        await super().delete(*args, **kwargs)
+        await self.search_index.delete_document(kwargs['id'])
+
+    async def patch(self, *args, **kwargs):
+        await super().patch(*args, **kwargs)
+        await self.index_document(kwargs['id'])
+
+
+class SearchIndexRequestHandler(operations_log.RequestHandlerMixin,
+                                base.AuthenticatedRequestHandler):
+    SQL = re.sub(r'\s+', ' ', """\
+        SELECT id
+          FROM v1.operations_log
+         ORDER BY id""")
+
+    async def get(self):
+        result = await self.postgres_execute(self.SQL)
+        ids = [row['id'] for row in result]
+        for operations_log_id in ids:
+            value = await models.operations_log(operations_log_id,
+                                                self.application)
+            await self.search_index.index_document(value)
```

Comment:
NIT - You don't need to create a new list containing the ids from the rows.
```suggestion
        for row in result:
            value = await models.operations_log(row['id', self.application)
            await self.search_index.index_document(value)
```